drf_standardized_errors/handler.py

```diff
@@ -136,5 +140,7 @@ def report_exception(
                     getattr(request, "path", ""),
                     response=response,
                     request=request,
-                    exception=exc,
+                    exception=original_exc[1],
```

Comment:
can you test what happens if we directly provide the original exception using `self.exc`? I want to see if that avoids the extra code to deal with the reference cycle issue. If that works, I'll merge the PR so that the fix is available on main. After that, you can open a new PR to add a test case for the issue